ui/tests/acceptance/enterprise-replication-test.js

```diff
@@ -88,6 +89,22 @@ module('Acceptance | Enterprise | replication', function(hooks) {
     await click('[data-test-secondary-add]');
 
     await pollCluster(this.owner);
+    // checks on secondary token modal
+    assert.dom('#modal-wormhole').exists();
+
+    const date = new Date();
```

Comment:
The point of this test is to make sure:
1: the modal is rendering the date in the correct format
2: that the time of adding the default 30 min matches to the generated token.  Adding 30 min to the the current timestamp vs decoding the token could result in a 1-2 second difference because "computer things".  Thus, I check the strings are the same up until the seconds.  

I think these checks are helpful and cover a lot of functionality, but if you can think of ways to make them easier to read or clean up, I'm all ears 👂.

Reply:
one thing that'd help with making the tests easier to read would be moving this to its own test. in an ideal scenario testing that the modal shows the right stuff would be inside a component test, but in lieu of that i think this should be kept separate from the rest of this test since this test is already super long (and really ought to be broken up 😅). 

whaddya think about moving this to a separate test for just adding a secondary using this new flow? 

and there's a few small naming nits that might help too. up to you!

Reply:
I think that's a great idea.  I'll move it and work on some of the naming.